### PR TITLE
OPGOPS-2062 Jenkins2 

### PR DESCRIPTION
**Pull request by carinadigital**
Upgrade to jenkins 2.60.1 LTS with updated plugins
Add confluence plugin
jenkins-slave instance containers now launch with same workspace inside or outside the containers.

### CODE REVIEW

File: jenkins2/Dockerfile [hunk 1]
```diff
@@ -106,14 +106,74 @@ COPY install-plugins.sh /usr/local/bin/install-plugins.sh
...
+  workflow-api:2.16 \
+  workflow-basic-steps:2.5 \
+  workflow-cps-global-lib:2.8 \
+  workflow-cps:2.33 \
+  workflow-durable-task-step:2.11 \
+  workflow-job:2.11 \
+  workflow-multibranch:2.13 \
+  workflow-scm-step:2.4 \
+  workflow-step-api:2.10 \
+  workflow-support:2.14
```
Comment (Azulinho) [inline jenkins2/Dockerfile:184]: can you add the confluence plugin ?
I think it was merged into the old jenkins after your started this work

Comment (carinadigital) [reply jenkins2/Dockerfile:184]: This one https://wiki.jenkins.io/display/JENKINS/Confluence+Publisher+Plugin ?

Comment (Azulinho) [reply jenkins2/Dockerfile:184]: ya

**Review by Azulinho — APPROVED** [review]

**Comment by carinadigital** [comment]
Tested building here https://jenkins.service.opg.digital/job/OPG/job/docker-build-base/247/

**Review by tim-mcarthur-moj — APPROVED** [review]

**Pull request merged**